doc/productivity.md

```diff
@@ -46,38 +46,33 @@ Note: base_dir is required for ccache to share cached compiles of the same file
 
 You _must not_ set base_dir to "/", or anywhere that contains system headers (according to the ccache docs).
 
-### Disable features with `./configure`
+### Disable features when generating the buildsystem
 
-After running `./autogen.sh`, which generates the `./configure` file, use `./configure --help` to identify features that you can disable to save on compilation time. A few common flags:
+During generating the buildsystem, only essential build options are enabled by default to save on compilation time.
 
-```sh
---without-miniupnpc
---without-natpmp
---disable-bench
---disable-wallet
---without-gui
-```
+Run `cmake -B build -LH` to see the full list of available options. GUI tools, such as `ccmake` and `cmake-gui`, can be also helpful.
 
-If you do need the wallet enabled, it is common for devs to add `--with-incompatible-bdb`. This uses your system bdb version for the wallet, so you don't have to find a copy of bdb 4.8. Wallets from such a build will be incompatible with any release binary (and vice versa), so use with caution on mainnet.
+If you do need the wallet enabled (`-DENABLE_WALLET=ON`), it is common for devs to use your system bdb version for the wallet, so you don't have to find a copy of bdb 4.8. Wallets from such a build will be incompatible with any release binary (and vice versa), so use with caution on mainnet.
 
-### Make use of your threads with `make -j`
+### Make use of your threads with `-j`
 
-If you have multiple threads on your machine, you can tell `make` to utilize all of them with:
+If you have multiple threads on your machine, you can utilize all of them with:
 
 ```sh
-make -j"$(($(nproc)+1))"
+cmake --build build -j "$(($(nproc)+1))"
```

Comment:
let's use the comment without mentioning nproc

Reply:
In this PR, I don't want introduce any changes that are unrelated to the migration to CMake.

Reply:
ok, but you've added an extra nproc, that's what I was commenting on

Reply:
The extra line has been dropped.